t/obs/systemsmanagement:Agama:Devel/files_iso.lst

```diff
@@ -1,12 +1,5 @@
-agama-live.aarch64-3.0.0-ALP-Build3.3.iso
-agama-live.ppc64le-3.0.0-ALP-Build3.3.iso
-agama-live.s390x-3.0.0-ALP-Build3.3.iso
-agama-live.x86_64-3.0.0-ALP-Build3.3.iso
-agama-live.aarch64-3.0.0-ALP-Playwright-Build3.3.iso
-agama-live.x86_64-3.0.0-ALP-Playwright-Build3.3.iso
-agama-live.aarch64-3.0.0-openSUSE-Build3.3.iso
-agama-live.ppc64le-3.0.0-openSUSE-Build3.3.iso
-agama-live.s390x-3.0.0-openSUSE-Build3.3.iso
-agama-live.x86_64-3.0.0-openSUSE-Build3.3.iso
-agama-live.aarch64-3.0.0-openSUSE-Playwright-Build3.3.iso
-agama-live.x86_64-3.0.0-openSUSE-Playwright-Build3.3.iso
+agama-installer-openSUSE.aarch64-9.0.0-openSUSE-Build9.35.iso
```

Comment:
build 9.35 is from 31st July and has the old name, we should sync:
`agama-installer.x86_64-9.0.0-openSUSE-Build8.7.iso`
We might need something like this:
`agama-installer\..*-openSUSE-Build.*iso$` in `<hdd` tag.